clustering/ejb/WFLY-14359_Create_distributable-ejb_subsystem.adoc

```diff
@@ -0,0 +1,170 @@
+= WFLY-14953 Create distributable-ejb subsystem
+:author:            Richard Achmatowicz
+:email:             rachmato@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The Undertow and EJB subsystems depend on certain abstractions provided by the clustering subsystems in order to support
+clustered deployments.
+
+Hard-coding clustering subsystem dependencies within the EJB and Undertow subsystems creates a tight-coupling between
```

Comment:
Hard-coding clustering subsystem dependencies within the EJB ~~and Undertow~~ subsystem~~s~~ creates a tight-coupling between